FUNDING_SPLITS.csv

```diff
@@ -1,11 +1,11 @@
 ethereum_address,github_username,impact_percentage
-0x5D388Ec24Cc2C0C77458338696aa63AFB706A7b1,nya-elimu,51.02
-0xd46Cc93E3eE6a4fb532D9B48E95be7eD8f8f1DA0,jo-elimu,38.77
-0x3965d420cde24fe00f742cb31257ca90e04556a7,tomaszsmy,1.20
-0x819e4F5E912abb36c8f6c03790e7508e03cc37a3,UmenR,1.12
-0x5367a6d4F5991327Bb20466684790e394bE000dE,Shubham-Vishwakarma,0.76
-0x0000000000000000000000000000000000000000,aschworer,0.73
-0x6321286F9B73f427C72e1f9F1bC6b3d25eF06605,hhio618,0.57
-0x0000000000000000000000000000000000000000,urosht,0.57
-0x4709ebf314c6492d57f4c6d4f57357d5b2bb074e,vrudas,0.56
-0x914dBEe36BcAb63B2D15fAED08839Be43797b421,watr-sheep,0.38
+0x5D388Ec24Cc2C0C77458338696aa63AFB706A7b1,nya-elimu,49.39
+0xd46Cc93E3eE6a4fb532D9B48E95be7eD8f8f1DA0,jo-elimu,39.00
+0x3965d420cde24fe00f742cb31257ca90e04556a7,tomaszsmy,1.14
+0x819e4F5E912abb36c8f6c03790e7508e03cc37a3,UmenR,1.08
+0x342FaeB14127A79735144bbdC772AbB6b1bBe343,coderabbitai,1.05
+0x5367a6d4F5991327Bb20466684790e394bE000dE,Shubham-Vishwakarma,0.73
+0x0000000000000000000000000000000000000000,aschworer,0.70
+0x0000000000000000000000000000000000000000,venkatesh2k3,0.58
```

Comment:
0x513DceE2929a6e0CD115fb65Ec926d6569d98aff

Reply:
> 0x513DceE2929a6e0CD115fb65Ec926d6569d98aff

@venkatesh2k3 Added it for you: https://github.com/elimu-ai/webapp/commit/eb36baa6805ef80b5a350950569b725c45335ff6

Reply:
Thank you @nya-elimu